fe/fe-core/src/main/java/com/starrocks/metric/MetricCalculator.java

```diff
@@ -127,5 +128,9 @@ private void update() {
             MetricRepo.GAUGE_QUERY_LATENCY_P99.setValue(0.0);
             MetricRepo.GAUGE_QUERY_LATENCY_P999.setValue(0.0);
         }
+
+        if (Config.enable_routine_load_lag_metrics)  {
```

Comment:
Why not make the async fetch the offset lags the only way？

Reply:
Later, Offset information support is obtained from the local cache as much as possible, instead of directly accessing Kafka, this can be turned on by default.Otherwise, the request to get kafka offset is a bit too much.